extensions/mute-microphone/src/shared/utils.ts

```diff
@@ -15,8 +20,14 @@ export async function toggleSystemAudioInputLevel(currentAudioInputLevel: number
 
   execFileSync(scriptPath, ["set", newLevel.toString()]);
 
-  const message = newLevel == 0 ? "Audio input muted" : "Audio input unmuted";
-  await showHUD(message);
+  if (newLevel == 0) {
+    toast.title = "Mic muted";
+    toast.style = Toast.Style.Failure;
+  } else {
+    toast.title = "Mic unmuted";
+    toast.style = Toast.Style.Success;
+  }
```

Comment:
Audio input is not only the microphone, so we should keep the original messages.

```suggestion
  if (newLevel == 0) {
    toast.title = "Audio input muted";
    toast.style = Toast.Style.Failure;
  } else {
    toast.title = "Audio input unmuted";
    toast.style = Toast.Style.Success;
  }
```